fuse/server.go

```diff
@@ -475,6 +476,44 @@ func (ms *Server) InodeNotify(node uint64, off int64, length int64) Status {
 	return result
 }
 
+// InodeNotifyStoreCache tells kernel to store data into inode's cache.
+//
+// This call is similar to InodeNotify, but instead of only invalidating a data
+// region, it gives updated data directly to the kernel.
+func (ms *Server) InodeNotifyStoreCache(node uint64, offset int64, data []byte) Status {
+	if !ms.kernelSettings.SupportsNotify(NOTIFY_STORE) {
+		return ENOSYS
+	}
+	if len(data) > math.MaxUint32 {
```

Comment:
Hm, instead of documenting, we could better split incoming data in max 4G size and issue several NotifyStoreCache calls. I will change to that.

However I observed that in many other places go-fuse silently converts e.g. int to uint32 and that can break on data with size > 4G. For example:

https://github.com/hanwen/go-fuse/blob/8ccb625b/fuse/request.go#L219
https://github.com/hanwen/go-fuse/blob/8ccb625b/fuse/opcode.go#L209
etc

About the change itself - I'm not sure how better to test - it needs at least 4GB of RAM in kernel pagecache, and that is not the case on my working station. Hopefully it is ok with just a careful for loop.